### DISCUSSION TITLE
How to implement virtual scrolling with dynamic and unpredictable height of list items

### DISCUSSION THREAD

**Discussion by jiaozhiye** [TanStack/virtual#891] [General]
My scenario is this: the height of each list item is not fixed and unpredictable. The height is determined by the amount of text inside. How can I use `useVirtualizer`? Can you help me and give me a demo?

**Comment by piecyk** [comment]
@jiaozhiye in this case you can use dynamic example https://tanstack.com/virtual/latest/docs/framework/react/examples/dynamic
